conan/tools/sbom/cyclonedx.py

```diff
@@ -3,6 +3,28 @@
 def cyclonedx_1_4(graph, name=None, add_build=False, add_tests=False, **kwargs):
     """
     (Experimental) Generate cyclone 1.4 sbom with json format
+
+    Generate a CycloneDX 1.4 document as a string from the given dependency graph.
```

Comment:
I think this would be more precise?
```suggestion
    Creates a CycloneDX 1.4 Software Bill of Materials (SBOM) from a given dependency graph.
```